### ISSUE TITLE
Feature request: Enable it by default

### ISSUE THREAD

**Issue by FrostShock** [Dridzt/AutoProfit#1]
I have tried AutoProfit but it is not enabled by default. That means, on each character/account where I want to use it, I have to enable it first.
There are other addons that also need to be enabled first, like BestBuff.

If you have to enable 30 addons on each new character / bank alt that you make, that's a big waste of time.

I am installing this addon because I want to use it, I really don't need to be disabled by default.

Because this, atm I am using GrayAutoSell

**Comment by Road-block** [comment]
The addon is enabled by default. 
I don't know what client / realm you are playing on and if it has non-blizzlike / custom features but this is not the addon's fault.  

For an addon to be disabled by default it has to have a tag in its .toc that reads `## DefaultState: disabled`
You can check [AutoProfit.toc](https://github.com/Road-block/AutoProfit/blob/master/AutoProfit.toc) yourself and you will find no such tag.

**Comment by FrostShock** [comment]
Sorry, probably "enabled" is not the right word.
Yes, it is loaded at login.
But the automatic selling is turned off by default
I have to type "/autoprofit auto"
And then I get: Autoprofit: automatic selling on.
Just like in Bestbuff, I have to type "/bestbuff" - then check "Enable" in order to actually make use of it.

If you install and load addons, you do that to use them right away, and you do not want to spend time tell to configure addons to actually make use of them.

I just noticed it has a button (at vendor) for selling all junk but I had to type "/autoprofit auto" in order to make it sell automatically, without the need to push the button.

I am playing on Elysium
